src/ir/table-utils.h

```diff
@@ -47,6 +48,48 @@ struct FlatTable {
   }
 };
 
```

Comment:
Should FlatTable go in this new namespace too?

Reply:
Personally I think it's excessive to add it - it's name already implies what type of thing it does. But I don't feel strongly. What do you think?

Reply:
Just seems odd to have a TableUtils that is declared in `table-utils.h` but not all the stuff in that header ends up in the namespace.  No need to hold up this PR on that though.  I don't feel super strongly about it.

Reply:
Fair enough, if it seems odd to you, I'll change that.